label_studio/users/api.py

```diff
@@ -48,8 +48,8 @@ def avatar(self, request, pk):
     @swagger_auto_schema(
         tags=['Users'],
         operation_summary='Save user details',
```

Comment:
As an administrator, I get an error that I don't have permissions to make these changes on a different user, so then who does have permissions?